canonical-web-and-design/ubuntu.com
# Rename camel case JS files

## Done

- Renamed camel case JS files to be hyphenated
- Also updated eslint so it doesn't conflict with prettier
- Removed `stickySidebar` as it didn't work
- Removed link to `stickyNav` on the pages that weren't using it

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run the site and check that there are no errors


## Issue / Card

Fixes #6907
